#103 updated parsePluginYamlFileTest() to text output check output to…

mikepain opened the pull request:
… System.out

Hi,  I'm new to open source development so have stuck a little toe in with what I see as this simple change. 

What I understand from the #103 is that all calls to System.out should be tested with SystemOutRule. I could only find one instance so have updated that.

Cheers

Mike

jetersen commented on plugin-management-library/pom.xml:
```diff
@@ -55,6 +55,13 @@
         </dependency>
+        <!-- https://mvnrepository.com/artifact/com.github.stefanbirkner/system-rules -->		<dependency>
+		    <groupId>com.github.stefanbirkner</groupId>
+		    <artifactId>system-rules</artifactId>
```
Properly better to use `system-lambda`

cc @stefanbirkner

stefanbirkner replied:
You're right. I added system-lambda in #123 to plugin-management-library.

stefanbirkner commented:
@jetersen I looked at the test and found out that the assertion is already correct. The `System.out.println` is some debug code that should be removed. Do you want to create a PR that removes this statement?

mikepain commented:
> @jetersen I looked at the test and found out that the assertion is already correct. The `System.out.println` is some debug code that should be removed. Do you want to create a PR that removes this statement?

Is a PR a change request? If so if you can let me know when it is created & the number & I will make the change

jetersen commented:
#123 is now merged so you should be able to merge or rebase with upstream master :)

mikepain commented:
> #123 is now merged so you should be able to merge or rebase with upstream master :)

Hi @jetersen ,

I've not used git /Github with a team before & need a little help please.

My understanding is that  I need to rebase my forked 'mikepain/plugin-installation-manager-tool' master with with 'jenkinsci/plugin-installation-manager-tool' but I can't see how to do this.

This is what I have done
1. I have re-pulled my original fork - 'git pull  mikepain/plugin-installation-manager-tool master'
2. I then tried to rebase my fork using - 'git rebase master'  but it is saying it is up to date which makes sense as I just pulled it.

Thanks for your help

Mike

mikepain commented:
> #123 is now merged so you should be able to merge or rebase with upstream master :)

I got caught up in a couple of merge issues which I've not had to resolve with git before. After a bit of a struggle, I gave up and deleted the fork & re-created the fork  'mikepain/plugin-installation-manager-tool'. I then removed the single line. Please review

timja commented:
could you please resolve the conflict

mikepain commented:
> could you please resolve the conflict

Hi @timja 

I'm a developer who hasn't done any development for 10 years & I'm new to using GitHub. I would like to sort this out but I'm not sure what I needed to do.

This is what I have done so far (I thought I had chosen a very simple change to understand how GitHub works but I have got out of my depth).

1. I created a new fork to make the request change (#103) ,I made the changes & committed the change to GitHub but missed some code layout errors & this is the error showing in the build above.
2. I tried to fix the build but had to merge my fork but I struggled to complete the merge using Eclipse. 
3. Because I was struggling with merge & the requirement changed to just a line delete,I deleted my original fork & took a new fork from the latest code. 
4. I then removed the one line & tried to update #103 so it could be reviewed. The new code is here https://github.com/mikepain/plugin-installation-manager-tool & have asked for help but I've had no response. 

Can you advise what I need to do, please

Thanks

Mike

timja commented:
I would create a new Pull request based off of the current master branch

It should use system-lambda as that’s what we’ve changed over to using recently.

mikepain commented:
Replaced by #150 so closing

> I would create a new Pull request based off of the current master branch
> 
> It should use system-lambda as that’s what we’ve changed over to using recently.

Hi @timja, 

thanks for your help. I created #150 & it appeared to build OK. Should I close this PR now & reference 150 in it??

Mike

timja commented:
Yes thanks, I’ll take a look into it later on and double check it’s not required

mikepain commented:
Closing PR as replaced by #150
